hoodie-client/src/main/java/com/uber/hoodie/index/bloom/BucketizedBloomCheckPartitioner.java

```diff
@@ -139,11 +140,10 @@ public int numPartitions() {
 
   @Override
   public int getPartition(Object key) {
-    String[] parts = ((String) key).split("#");
-    String fileName = parts[0];
+    final String[] parts = ((String) key).split("#", 2);
     final long hashOfKey = Hashing.md5().hashString(parts[1], StandardCharsets.UTF_8).asLong();
-    List<Integer> candidatePartitions = fileGroupToPartitions.get(fileName);
-    int idx = (int) Math.floorMod(hashOfKey, candidatePartitions.size());
+    final List<Integer> candidatePartitions = fileGroupToPartitions.get(parts[0]);
+    final int idx = (int) Math.floorMod(hashOfKey, candidatePartitions.size());
```

Comment:
I can back these changes out, no need to update this file here

Reply:
Thats ok